birdhouse/components/stac/service-config.json.template

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
+  "name": "stac",
+  "keywords": [
+    "catalog"
+  ],
+  "description": "Searchable spatiotemporal metadata describing climate and Earth observation datasets.",
+  "links": [
+    {
+      "rel": "service",
+      "type": "application/json",
+      "href": "https://${PAVICS_FQDN_PUBLIC}/stac/"
+    },
+    {
+      "rel": "service-doc",
+      "type": "text/html",
+      "href": "https://github.com/radiantearth/stac-api-spec"
+    },
+    {
+      "rel": "service-desc",
+      "type": "application/yaml",
+      "href": "https://raw.githubusercontent.com/radiantearth/stac-api-spec/main/core/openapi.yaml"
```

Comment:
I think it would be better actually to refer to a pinned version (release/tag). 
https://github.com/radiantearth/stac-api-spec/tree/v1.0.0 should be good. 
We should probably make this a configurable env-var with the corresponding app tag for the docker image. 
Right now, `crim-ca/stac-app:main` is used as well: 
https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/components/stac/docker-compose-extra.yml#L13

Reply:
I agree about the pinned version. I didn't quite understand your second comment..

Reply:
@fmigneault can this be addressed in a different PR? It's a good point but probably should be addressed later when we actually start using a version other than `crim-ca/stac-app:main`

Reply:
Yes. I agree to move it to another PR while addressing the reference image.